README.md

```diff
@@ -1,48 +1,28 @@
 # Primitives
 
-## bridge_podnet
-Primitive for Building Public subnet bridge on PodNet using netplan service.
+## net_main
```

Comment:
Is the name of the primitive to be `netmain` or `net_main`?

Reply:
I as remember we suggested to use <primitive>[_<hardware>] but somehow we started using <primtive>[hardware].